DuckDuckGo/HomeMessageView.swift

```diff
@@ -37,29 +37,29 @@ struct HomeMessageView: View {
     let viewModel: HomeMessageViewModel
     
     var body: some View {
-        VStack(spacing: Const.Spacing.titleAndSubtitle) {
-            HStack(alignment: .top) {
-                Spacer()
-                VStack(spacing: 0) {
-                    topText
-                    image
-                    title
+        ZStack {
+            closeButtonHeader
+                .offset(x: Const.Padding.buttonHorizontal, y: -Const.Padding.buttonHorizontal)
+            VStack(spacing: Const.Spacing.titleAndSubtitle) {
+                HStack(alignment: .top) {
```

Comment:
Does it need to be inside this HStack?

Reply:
Good catch! Yep, that HStack was no longer needed and I've removed, thanks!